### ISSUE TITLE
EvilCraft does not generate deepslate dark ore, and rare normal dark ore.

### ISSUE THREAD

**Issue by uhloin** [rubensworks/EvilCraft#1018]
Minecraft 1.20.1
forge-1.20.1-47.1.81-installer.jar 
CyclopsCore-1.20.1-1.18.14.jar 
EvilCraft-1.20.1-1.2.35.jar 

Book says dark ore may be found on the same height as redstone. In 1.19 it was true.
1.20.1: no dark ores generated at levels -64..0, no deepslate dark ore at all.
Normal dark ore become too rare for fair play, about 5 blocks per chunk at levels 0+

I tried different versions of EC/CC for 1.20.1 since initial until current with only 1 additinal mod (Mekanism) to detect ores with Digital Miner.

**Comment by rubensworks** [comment]
Last time I checked ore spawned at usual rates.

Are you using a modpack where the pack creator may have changed spawn rates?
Or are you using a mod that might modify spawn rates?
Could you check without any other mods installed?

**Comment by uhloin** [comment]
Yes, I use other mods, but for this test I've reset minecraft to initial state + forge + evilcraft + cyclopscore + mekanism, nothing more. Mekanism's miner cannot find any dark ore below level 0. If you know how to check ore generation without Mekanism, say me, I'll try. It's still possible to search ore above level 0 with some cheats or mods, but probably this is not way you expect :)

**Comment by tonby** [comment]
I also tested in the smallest environment and found no Deepslate Dark Ores, and only a few (stone) Dark Ores.
Additionally, I observed Dark Ores appearing at Y=34, a level where Redstone doesn't usually spawn.
There may be an issue with the height of Dark Ore generation.

Versions:

Minecraft: 1.20.1
Forge: 47.2.0
CyclopsCore: 1.20.1-1.18.14
EvilCraft: 1.20.1-1.2.35
![2023-12-22_22 57 29](https://github.com/CyclopsMC/EvilCraft/assets/37474828/4184ea2e-8b76-4d67-93fc-822db1eec40b)
